cloud_service/upgrading-cloud/upgrade-cloudsvc-roxctl.adoc

```diff
@@ -0,0 +1,68 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="upgrade-cloudsvc-roxctl"]
+= Manually upgrading secured clusters in RHACS Cloud Service by using the `roxctl` CLI
```

Comment:
From the [OCP documentation guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc): "Do not use backticks or other markup in assembly or module headings."
```suggestion
= Manually upgrading secured clusters in RHACS Cloud Service by using the roxctl CLI
```